kahuna/public/js/search/results.html

```diff
@@ -15,7 +15,22 @@
         <div class="results-toolbar"
             ng:class="{'results-toolbar--selection-mode': ctrl.metadataPanelAvailable}">
 
-            <div class="results-toolbar-item results-toolbar-item--static results-toolbar-item--left
+            <div ng-show="false"
+                class="results-toolbar-item
+                results-toolbar-item--left
+                clickable"
+                 ng:click="ctrl.toggleCollectionsPanel()">
```

Comment:
If an element can be clicked, please use a clickable control for accessibility reasons and making things more obvious. Here `button` would make most sense.